netbox/dcim/choices.py

```diff
@@ -1374,6 +1382,19 @@ class PortTypeChoices(ChoiceSet):
                 (TYPE_SPLICE, 'Splice'),
             ),
         ),
+        (
+            _('USB'),
+            (
+                (TYPE_USB_A, 'USB-A'),
+                (TYPE_USB_B, 'USB-B'),
+                (TYPE_USB_C, 'USB-C'),
+                (TYPE_USB_MINI_A, 'USB-Mini-A'),
+                (TYPE_USB_MINI_B, 'USB-Mini-B'),
+                (TYPE_USB_MICRO_A, 'USB-Micro-A'),
+                (TYPE_USB_MICRO_B, 'USB-Micro-B'),
+                (TYPE_USB_MICRO_AB, 'USB-Micro-AB'),
```

Comment:
The human-friendly labels for these types should match what is already defined for console port types in the interest of consistency.

Reply:
I will resolve/adjust and resubmit.  Thank you.